### ISSUE TITLE
MirTop Broken (different error 🤦🏻 )

### ISSUE THREAD

**Issue by apeltzer** [ewels/MultiQC#1778]
### Description of bug

![image](https://user-images.githubusercontent.com/2359510/195032081-2012088e-242a-4a53-b63a-ae5c66c30670.png)

### File that triggers the error

[full_mirtop_stats_mirtop_mirsum.log](https://github.com/ewels/MultiQC/files/9752795/full_mirtop_stats_mirtop_mirsum.log)

### MultiQC Error log

```console
MultiQC 1.13, only happens when the data being processed is probably only few reads, thus mirtop seems to skip outputting certain stats. Upstream bugfix maybe an option too?
```

**Comment by apeltzer** [comment]
@ErikDanielsson

**Comment by ErikDanielsson** [comment]
Sorry forgot the negation :facepalm:

**Comment by apeltzer** [comment]
Ah, was already excited. Opening an issue on MirTop itself now to describe the problem. I think the tool should just generate _always_ the same output fields.

**Comment by ErikDanielsson** [comment]
What I meant is that I was only working on MultiQC for the summer, now I'm back to studying

**Comment by apeltzer** [comment]
Yes, just added an issue on MirTop as I'm also thinking it could be something to tackle there insteadd of implementing workarounds in MultiQC. Thanks erik!

**Comment by ewels** [comment]
Would be kind of nice to handle it here, even if it's handled upstream - for backwards compatibility of previous versions.

**Comment by ewels** [comment]
Fixed in eaa850f5097253f3575ea5490e4a45ea33ad8b9e ✅
